AEPIdentity/Sources/IdentityState.swift

```diff
@@ -283,6 +283,30 @@ class IdentityState {
         }
     }
 
+    /// Clears all identities and regenerates a new ECID value.
+    /// Saves identities to persistence and creates a new shared state and dispatches a new` resetComplete` event after operation completes.
+    /// - Parameters:
+    ///   - event: event which triggered the reset call
+    ///   - createSharedState: function which creates new shared states
+    ///   - eventDispatcher: function which dispatches a new `Event`
+    func resetIdentifiers(event: Event,
+                          createSharedState: ([String: Any], Event) -> Void,
+                          eventDispatcher: (Event) -> Void) {
+        // clear the properties
+        identityProperties = IdentityProperties()
```

Comment:
Actually, this will reset the stored privacy status to unknown. Maybe we should check if privacy is optedout and if so, simply abort?